{{cookiecutter.project_folder}}/requirements-dev.txt

```diff
@@ -1,7 +1,8 @@
-autopep8
 attrs >=17.4
+autopep8
```

Comment:
should we add `black`?

Reply:
yes, absolutely, I forgot about it :P added now